.github/workflows/photonvision-docs.yml

```diff
@@ -0,0 +1,45 @@
+name: PhotonVision Sphinx Documentation
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'docs/**'
+      - '.github/**'
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'docs/**'
+      - '.github/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
```

Comment:
3.9 is specified here but 3.11 is specified below

Reply:
yeah ez fix. also fixed it so we are no longer cloning the docs repo anywhere